Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM romeoz/docker-apache-php:7.3
+
+RUN a2enmod headers && a2enmod rewrite && a2enmod expires
+
+RUN apt-get update \
+    && apt-get install -y openssl php7.3-curl php7.3-gd php7.3-mbstring php7.3-mysql php7.3-soap php7.3-gmp \
+    && rm -rf /var/lib/apt/lists/* 
+
```

Comment:
You would need to add the Mysql-Server installation.

Reply:
You should already have a mysql running anyway right?

Reply:
> You should already have a mysql running anyway right?

Yes, the issue is whether you can connect the docker to an external MySQL. Because if I remember correctly, in the docker-compose.yml there were some rules about network.

Reply:
Only if your mysql server is running via Docker, should be in the same network. But that's an edge case and there is now way of knowing what network name the user is using, so we can't ship that by default. Outer network access should work flawlessly.